rustls-libssl/admin/matrix.py

```diff
@@ -0,0 +1,240 @@
+#!/usr/bin/env python3
+
+# from openssl 3.0.0
```

Comment:
nit: maybe "from openssl 3.0.0" -> "from openssl 3.0.0 util/libssl.num"

Saves the next person having to find it in the src tree.